recipes/Text_to_Shell_Exec/Text_to_Shell_Exec.ipynb

```diff
@@ -162,20 +178,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "def chat(prompt: str, \n",
+    "def chat(prompt: str,\n",
     "         system_prompt:str = default_system_prompt,\n",
-    "         examples: str = examples, \n",
-    "         model: str ='granite-code:8b') -> str:\n",
+    "         examples: str = examples,\n",
+    "         model: str ='granite-code:3b') -> str:\n",
```

Comment:
Noting the downstep here as well.  An argument for using Replicate as the default model platform would be that we can leave in the large models as defaults.